### DISCUSSION TITLE
how make the audio eac3to to qaac in automatic

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#990] [Q&A]
i use staxrip 1.7.0.0 and i want to make audio setting in staxrip 1.7.0.0 as staxrip 2.14.0

this is picture i use
![1](https://user-images.githubusercontent.com/79289286/229246317-2cda8572-c28b-438f-b6cc-5e818eaf35ac.PNG)

this is picture i want in staxrip 1.7.0.0
![2](https://user-images.githubusercontent.com/79289286/229246322-867dc864-ffab-445c-88d1-0d7506d9f7cd.PNG)

**Comment by jessielw** [comment]
Why can't you update to 2.14.0?

**Reply by mr-qansuh** [reply]
Because version 1.7 .0.0 is faster in merging than  2.14.0 and and always use all of cpu speed

**Reply by jessielw** [reply]
In what encoder?

**Reply by mr-qansuh** [reply]
x264
